tests/mlflow/test_kedro_pipeline_model.py

```diff
@@ -85,6 +85,9 @@ def test_model_packaging(tmp_path, pipeline_ml_obj):
     )
     assert loaded_model.predict(1) == {"predictions": 2}
 
+    if mlflow.active_run():
+        mlflow.end_run()
+
```

Comment:
This is very likely something we can do for all tests. Pytest have a way to call some cleanup code after each tests, and we must force all active mlflow runs (especially when tests fails)